code/modules/organs/cybernetics/upgrades/cybernetics.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/cybernetics
+	//This thing doesn't actually exist, just something for inheritance
+	var/required_type //If the upgrade is only for a specific organ, if not then keep it empty
+
+/obj/item/cybernetics/afterattack(var/obj/item/O)
+	if(!Adjacent(usr))
+		return
+
+	if(istype(O, /obj/item/organ))
+		var/obj/item/organ/I = O
+		if(!(I.robotic == 2))
+			to_chat(usr, "\the [O] isn't robotic.")
+			return
+		to_chat(usr, "DEBUG - Succesfull application of [src] to [I].")
```

Comment:
That's why I write "HITLERS:" instead of "debug" or "todo" or anything you could easily miss